keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -774,26 +751,19 @@ are missing a bunch of machinery and tooling and can't do that now.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-<!--
-Even if applying deprecation policies, they may still surprise some users.
--->
+No
 
 ### Monitoring Requirements
 
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
--->
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-<!--
-Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-checking if there are objects with field X set) may be a last resort. Avoid
-logs or events for this purpose.
--->
+A new group of metrics are added for the new Cluster IP allocators
+1. IP allocator pkg/registry/core/service/ipallocator/metrics.go
+2. IP repair loop pkg/registry/core/service/ipallocator/controller/metrics.go
+
+Users can also obtain the `ServiceCIDR` and `IPAddress` objects that are only available
+if the feature is enabled.
```

Comment:
Are there any other data observable by the cluster admin, so that one can clearly say this functionality is or is not working in the cluster?

Reply:
I'd probably suggest writing a statement building around the metrics you provide below.

Reply:
expanded now with more detailed instructions, PTAL